### PR TITLE
Update JUnit tests

### PR DESCRIPTION
**Pull request by ryan1604**

### CODE REVIEW

File: src/test/java/seedu/financialplanner/commands/BudgetCommandTest.java [hunk 1]
```diff
@@ -34,4 +35,88 @@ public void testUpdateBudget() throws FinancialPlannerException {
         assertEquals(1500, Budget.getInitialBudget());
         assertEquals(1500, Budget.getCurrentBudget());
     }
+
+    @Test
+    @Order(3)
+    public void testResetBudget() throws FinancialPlannerException {
+        BudgetCommand testBudget = new BudgetCommand(Parser.parseRawCommand("budget reset"));
+        AddCashflowCommand testExpense = new AddCashflowCommand(Parser.parseRawCommand("add expense /a 50 /t dining"));
```
Comment (NeoMinWei) [inline src/test/java/seedu/financialplanner/commands/BudgetCommandTest.java:43]: maybe can add a parsertest class in the future to ensure that it is bug free

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: src/test/java/seedu/financialplanner/commands/BudgetCommandTest.java [hunk 2]
```diff
@@ -34,4 +35,88 @@ public void testUpdateBudget() throws FinancialPlannerException {
...
+        assertEquals(1000, Budget.getInitialBudget());
+        assertEquals(1000, Budget.getCurrentBudget());
+    }
+
+    @Test
+    @Order(4)
+    public void testDeleteBudget() throws FinancialPlannerException {
+        BudgetCommand testBudget = new BudgetCommand(Parser.parseRawCommand("budget delete"));
+        testBudget.execute();
+        assertFalse(Budget.hasBudget());
```
Comment (wwweert123) [inline src/test/java/seedu/financialplanner/commands/BudgetCommandTest.java:61]: Good use of negative testing

File: src/test/java/seedu/financialplanner/commands/BudgetCommandTest.java [hunk 3]
```diff
@@ -34,4 +35,88 @@ public void testUpdateBudget() throws FinancialPlannerException {
...
+    }
+
+    @Test
+    @Order(5)
+    public void testInvalidCommandFormat_throwsException() throws FinancialPlannerException {
```
Comment (wwweert123) [inline src/test/java/seedu/financialplanner/commands/BudgetCommandTest.java:66]: Good unit testing involving exceptionhandling

File: src/test/java/seedu/financialplanner/list/BudgetTest.java [hunk 4]
```diff
@@ -70,4 +73,13 @@ public void testDeleteBudget() {
         assertEquals(0, Budget.getCurrentBudget());
         assertFalse(Budget.hasBudget());
     }
+
+    @Test
+    @Order(8)
```
Comment (wwweert123) [inline src/test/java/seedu/financialplanner/list/BudgetTest.java:78]: Nice ordering of test cases

**Review by wwweert123 — COMMENTED** [review]
Very good!

**Pull request merged**